main.py

```diff
@@ -1,14 +1,17 @@
 from pyiron_base import Project
 
 pr = Project("some_project_name")
-job = pr.create.job.tensiletest("some_job_name", delete_existing_job=True)
+job = pr.create.job.TensileTest("some_job_name", delete_existing_job=True)
```

Comment:
Is there an option (instead of delete_existing) to redo the job only if dependencies (in particular script.py and potentially other files that scipt.py includes) have been modified.